modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp

```diff
@@ -96,7 +96,7 @@ renderResponse.setTitle(editable ? LanguageUtil.get(request, "edit-template") :
 								/>
 							</clay:alert>
 
-							<div class="mt-2">
+							<div class="mt-2" hidden>
```

Comment:
What do you think of `<div class="mt-2 d-none">`?

Reply:
Much better!!